packages/lesswrong/components/sunshineDashboard/ModeratorActions.tsx

```diff
@@ -329,9 +333,9 @@ export const ModeratorActions = ({classes, user, currentUser, refetch, comments,
         {user.sunshineFlagged ? <FlagIcon /> : <OutlinedFlagIcon />}
       </div>
     </LWTooltip>
-    {!user.needsReview && <LWTooltip title="Return this user to the review queue">
-      <VisibilityOutlinedIcon className={classes.modButton} onClick={handleNeedsReview}/>
-    </LWTooltip>}
+    <LWTooltip title="Return this user to the review queue">
+      <VisibilityOutlinedIcon className={classNames(classes.modButton, {[classes.disabledButton]: user.needsReview})} onClick={handleNeedsReview}/>
+    </LWTooltip>
```

Comment:
FYI this button isn't actually disabled (ex. it still adds an entry to the sunshine notes field)

Reply:
Yeah I think the `onClick` handler needs to account for that